.claude/commands/pm/issue-close.md

```
@@ -0,0 +1,107 @@
+---
+allowed-tools: Bash, Read, Write, LS
+---
+
+# Issue Close
+
+Mark an issue as complete and close it on GitHub.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)